godot-codegen/src/special_cases/special_cases.rs

```diff
@@ -108,6 +108,13 @@ pub fn is_godot_type_deleted(godot_ty: &str) -> bool {
         return true;
     }
 
+    // UniformSetCacheRD has been reintroduced in 4.3
+    // see: https://github.com/godotengine/godot/commit/5a988456
```

Comment:
Could you point to the PR rather than the commit here?

Reply:
done